.github/workflows/ci.yml

```diff
@@ -15,6 +15,15 @@ jobs:
 
     runs-on: ubuntu-latest
 
+    services:
+      mysql:
+          image: mysql:8
+          env:
+            MYSQL_ALLOW_EMPTY_PASSWORD: yes
+            MYSQL_DATABASE: laravel
+          ports:
+            - 3306
+          options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
```

Comment:
¿Para que esta opción?

Reply:
En este caso me basé en algunos tutoriales y mencionaban que era necesario ejecutar el health y en algunos intentos de correr el ci me sacaba el siguiente mensaje. "[error]Failed to initialize, mysql service is unhealthy." .